flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/detail/job-checkpoints-detail.component.html

```diff
@@ -81,6 +82,9 @@
             {{ checkPointDetail['tasks'][vertex.id]['end_to_end_duration'] | humanizeDuration }}
           </td>
           <td *ngIf="checkPointDetail['tasks'][vertex.id]['end_to_end_duration'] < 0">n/a</td>
+          <td>
+            {{ checkPointDetail['tasks'][vertex.id]['checkpointed_size'] | humanizeBytes }}
+          </td>
           <td>{{ checkPointDetail['tasks'][vertex.id]['state_size'] | humanizeBytes }}</td>
```

Comment:
I think instead of adding a new column, we could show both full and incremental size separated by `/` or in parenthesis. That would give a bit more space for future info, and **probably** be more readable.
WDYT?

Reply:
I think adding a new column could make user aware of this new feature. Moreover, changing this would make me update all pictures again, which is really annoying and I have already done this twice...